commands/utility/tonyQuote.js

```
@@ -0,0 +1,371 @@
+				{ name: 'Cost', value: `👑 ${TRIGGER_SUBMISSION_COST}`, inline: true },
+				{ name: 'Quote', value: `>>> "${quoteText}"`, inline: false },
+			)
+			.setFooter({ text: 'Please review this submission carefully.' })
+			.setTimestamp();
+
+
+		const result = db.prepare(`
+            INSERT INTO tony_quotes_pending (trigger_word, quote_text, user_id, approval_message_id, quote_type)
+            VALUES (?, ?, ?, ?, 'trigger')
+        `).run(triggerWord, quoteText, userId, approvalMessage.id);
+		const approvalMessage = await sendMessageToChannel(interaction.client, APPROVAL_CHANNEL_ID, approvalEmbed);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)